docs/resources/flexible_ip.md

```diff
@@ -0,0 +1,77 @@
+---
+page_title: "Scaleway: scaleway_flexible_ip"
+description: |-
+  Manages Scaleway Flexible IPs.
+---
+
+# scaleway_flexible_ip
+
+Creates and manages Scaleway flexible IPs.
+For more information, see [the documentation](https://developers.scaleway.com/en/products/flexible-ip/api).
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_flexible_ip" "main" {
+    reverse = "my-reverse.com"
+}
+```
+
+### With baremetal server
+
+```hcl
+data "scaleway_account_ssh_key" "main" {
+    name = "main"
```

Comment:
Can you put a really ECDSA key there? So that people can quickly identify where is the key